ner_v1/detectors/textual/name/name_detection.py

```diff
@@ -227,6 +288,176 @@ def context_check_botmessage(botmessage):
             True
         """
 
-        if "name" in botmessage:
+        if "name" in botmessage or u'नाम' in botmessage:
```

Comment:
Add `['name', 'नाम']` in constant file and check using 'in' condition

Reply:
Done